Changeing default --v_bitrate of recordmydesktop to 0 instead of 45000. 
Other values produce laggy and corrupted videos.--- a/src/plugins/recorder/recordmydesktop/recorditnow_recordmydesktop.kcfg
+++ b/src/plugins/recorder/recordmydesktop/recorditnow_recordmydesktop.kcfg
@@ -45,7 +45,7 @@
             <default>100</default>
     </entry>
     <entry name="__v_bitrate" type="Int">
-            <default>45000</default>
+            <default>0</default>
     </entry>
     <entry name="__s_quality" type="Int">
             <default>100</default>
--- a/src/plugins/recorder/recordmydesktop/ui/recordmydesktop.ui
+++ b/src/plugins/recorder/recordmydesktop/ui/recordmydesktop.ui
@@ -300,7 +300,7 @@
             </sizepolicy>
            </property>
            <property name="minimum">
-            <number>45000</number>
+            <number>0</number>
            </property>
            <property name="maximum">
             <number>2000000</number>
